cmd/util/ledger/migrations/staged_contracts_migration_test.go

```diff
@@ -1697,7 +1697,7 @@ func TestStagedContractsWithUpdateValidator(t *testing.T) {
 		require.Len(t, logWriter.logs, 1)
 		assert.Contains(t,
 			logWriter.logs[0],
-			"incompatible type annotations. expected `FungibleToken.Vault`, found `{FungibleToken.Vault}`",
+			"incompatible type annotations. expected [FungibleToken.Vault], found `{FungibleToken.Vault}`",
```

Comment:
The change is in a string literal, so should not be changed. It also refers to a Cadence type, and using brackets changes the meaning (array)